FWCore/Modules/src/GenericConsumer.cc

```diff
@@ -72,6 +75,12 @@ namespace edm {
                 edm::InputTag{branch.moduleLabel(), branch.productInstanceName(), branch.processName()});
           break;
 
+        case InProcess:
+          if (std::binary_search(processLabels_.begin(), processLabels_.end(), branch.moduleLabel()) or
+              std::binary_search(processLabels_.begin(), processLabels_.end(), kWildcard))
+            this->consumes(edm::TypeToGet{branch.unwrappedTypeID(), PRODUCT_TYPE},
```

Comment:
This needs
```suggestion
            this->consumes<edm::InProcess>(edm::TypeToGet{branch.unwrappedTypeID(), PRODUCT_TYPE},
```
right? (sorry, noticed only now)

Reply:
Good catch. Done.